api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/widgets_controller.rb

```diff
@@ -25,7 +25,12 @@ def index
   def create
     if widgets
       if @widget = widgets.create(widget_create_params)
-        MnoEnterprise::EventLogger.info('widget_create', current_user.id, 'Widget Creation', @widget)
+        if @widget.settings['organization_ids'].length >= 1
```

Comment:
There few operations like this one along the PR. 
Would not be more clear something like?
if you are expecting an array
```
if @widget.settings['organization_ids'].any?
```
or if you are expecting a string
```
if @widget.settings['organization_ids'].present?
```